target/linux/layerscape/patches-5.4/302-dts-0074-arm64-dts-fsl-ls1028a-Add-Felix-switch-port-DT-node.patch

```diff
@@ -48,55 +48,55 @@ Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
  			ethernet@0,4 {
  				compatible = "fsl,enetc-ptp";
  				reg = <0x000400 0 0 0 0>;
- 				clocks = <&clockgen 4 0>;
+ 				clocks = <&clockgen 2 3>;
  				little-endian;
  			};
 +			switch@0,5 {
 +				compatible = "mscc,felix-switch";
 +				reg = <0x000500 0 0 0 0>;
 +
 +				ports {
-+					#address-cells = <1>;
```

Comment:
Any reason for all those indent-changes? Same in the following patch?

Am I not seeing something here?